src/components/WaveDivider.js

```diff
@@ -1,13 +1,17 @@
-const WaveDivider = ({ color = 'white' }) => {
+const WaveDivider = ({ color = 'white', absolute }) => {
```

Comment:
maybe we should add a default value to make it a bit easier to infer what type we expect:
```suggestion
const WaveDivider = ({ color = 'white', absolute = false }) => {
```